drivers/serial/uart_rpi_pico.c

```diff
@@ -68,9 +70,14 @@ static int uart_rpi_init(const struct device *dev)
 		return ret;
 	}
 
+	/*
+	 * Can be used for resetting the UART device.
+	 * reset_toggle_dt(&config->reset);
```

Comment:
note: why is this commented out?

Reply:
Because the uart_init function from Pico SDK resets the uart device, after the implementation of a clock controller driver for Pico we can remove the uart_init call and use our clock and reset controller for this.

Reply:
if the driver is not using reset controller now, better remove it until it's possible.

Reply:
Okay, I removed it.